mlir/lib/Dialect/Tosa/IR/TosaOps.cpp

```diff
@@ -790,7 +880,7 @@ LogicalResult tosa::FullyConnectedOp::inferReturnTypeComponents(
   return success();
 }
 
-LogicalResult FullyConnectedOp::verify() { return verifyConvOp(*this); }
+LogicalResult FullyConnectedOp::verify() { return success(); }
```

Comment:
Would be good not to regress in the checking here, can some relevant checks be copied over? (worryingly no tests caught this 😅)